.github/workflows/build_ios.yml

```diff
@@ -40,10 +40,17 @@ jobs:
         service_account_email: ${{ secrets.GCS_EMAIL }}
         service_account_key: ${{ secrets.GCS_KEY }}
     
+    ############################################
+    # download credential files 
+    ############################################
+    - name: Download credential files
+      run: ./get-credentials.sh
+    
     ##############################################
-    # run everything we setup
+    # run tests, build android, build ios
     ##############################################
-    - run: gsutil cp gs://adventures-in-credentials/GoogleService-Info.plist ./ios/Runner/GoogleService-Info.plist
     - run: flutter pub get
     - run: flutter test
-    - run: flutter build ios --release --no-codesign
+    - run: flutter build appbundle -t lib/app/main.dart --build-number=$GITHUB_RUN_NUMBER
```

Comment:
I tried to compile the Android release version with this command and it returned this error:
```
Target file "lib/app/main.dart" not found.
```
I think that the path of the entry point should be `lib/main.dart` in our case (or we could remove the `-t` flag entirely since it defaults to `lib/main.dart` anyway).